python/tvm/driver/tvmc/compiler.py

```diff
@@ -82,6 +82,11 @@ def add_compile_parser(subparsers):
         help="path to an auto-tuning log file by AutoTVM. If not presented, "
         "the fallback/tophub configs will be used",
     )
+    parser.add_argument(
+        "--use-autoscheduler",
+        action="store_true",
+        help="use the autoscheduler to generate the compute schedules",
+    )
```

Comment:
You actually don't need users to tell you if auto_scheduler is used. Since AutoTVM and AutoScheduler have different log formats, you can simply use the following logic to know which one you should use:

```python
def is_auto_scheduler_log(log_file):
    try:
        auto_scheduler.load_records(log_file)
    except Exception:
        return False
    return True
```

Reply:
Very nice suggestion, thanks!